index.html

```diff
@@ -97,12 +87,23 @@ <h1>A broad selection of courses</h1>
 						published every month
 					</div>
 				</div>
+				<div class="courses-viewer">
+					<div class="categories"><b>python</b></div>
+					<div class="categories">Exel</div>
+					<div class="categories">Web Development</div>
+					<div class="categories">Java script</div>
+					<div class="categories">Data science</div>
+					<div class="categories">Aws</div>
+					<div class="categories">Drawing</div>
+				</div>
```

Comment:
You can also load this dynamically from the javascript, this way in the future if you want to make any change you will only update one place in the code and also can update the categories from the backend.